dataclients/kubernetes/kube.go

```diff
@@ -37,6 +37,8 @@ const (
 	ingressesNamespaceFmt         = "/apis/extensions/v1beta1/namespaces/%s/ingresses"
 	ingressClassKey               = "kubernetes.io/ingress.class"
 	defaultIngressClass           = "skipper"
+	ingressRouteIDPrefix          = "kube"
+	defaultEastWestDomainFmt      = "%s.%s.skipper.cluster.local"
```

Comment:
would it make sense to use go templating and require the string to be something like: `{{ .Name }}.{{ .Namespace }}.skipper.cluster.local`. This would allow having namespace before service name if desired and be easier to understand from a user point of view I think.

This idea is inspired by how we did it in external-dns: https://github.com/kubernetes-incubator/external-dns/blob/c0968ab857201b61a300aeb1a0b1b41a6f0daad3/pkg/apis/externaldns/types.go#L226

Reply:
discussed offline